src/java.xml/share/classes/javax/xml/catalog/CatalogFeatures.java

```diff
@@ -166,7 +166,9 @@
  *
  * <h2>Scope and Order</h2>
  * Features and properties can be set through the catalog file, the Catalog API,
- * system properties, and {@code jaxp.properties}, with a preference in the same order.
+ * system properties, and
+ * <a href="../../../module-summary.html#ConfigurationFile">configuration file</a>,
+ * with a preference in the same order.
```

Comment:
You may need to clarify the preference order a bit further, is is left-to-right or right-to-left in this list?